computers/paratera-th2/README

```diff
@@ -0,0 +1,4 @@
+#### NOTE
+
+The hostname of the computer maybe different depend on the proxy host provide by paratera.
+If the username is `<user>@<location>` then the user directory should be setting to `/PARA/<user>/aiida`. 
```

Comment:
I think this sentence can be deleted, since this happens automatically (right?).

Reply:
Sadly, no.
Since the default value of the workdir is `/scratch/<username>/aiida`. However, the username provided by paratera is something like `unkcpz@GZ` with an `@` symbol at the middle to (I guess) distinguish users from different areas.

Reply:
Oh sorry, I misread it.
That's interesting, I've never seen that ;-)